backend/test/invite_user_adm_handler_test.py

```diff
@@ -14,15 +14,24 @@
 
 from mock import patch
 
-def addAdminPermission(user, institution_key):
+def add_admin_permission(user, institution_key):
     user.add_permissions(permissions.DEFAULT_ADMIN_PERMISSIONS, institution_key)
 
-def hasAdminPermissions(user, institution_key):
+def add_super_user_permission(user, institution_key):
+    user.add_permissions(permissions.DEFAULT_SUPER_USER_PERMISSIONS, institution_key)
+
+def has_admin_permissions(user, institution_key):
     for permission in permissions.DEFAULT_ADMIN_PERMISSIONS:
         if not user.has_permission(permission, institution_key):
             return False
     return True
 
+def has_super_user_permissions(user, institution_key):
```

Comment:
This method does exactly the same thing as has_admin_permissions, it would be better to pass the permission type to be added per parameter.